### PR TITLE
add AntdOpenBrowser plugin

### PR DESCRIPTION
**Pull request by HenryTSZ**
Just place the contents of the Release zip in your %appdata%/FlowLauncher/Plugins folder and restart FlowLauncher.

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1621,5 +1621,19 @@
         "IcoPath": "https://cdn.jsdelivr.net/gh/JohnTheGr8/Flow.Plugin.RegJump@main/src/icon.png",
         "DateAdded": "2023-06-03T09:54:28Z",
         "LatestReleaseDate": "2023-05-30T22:07:24Z"
+    },
+    {
+      "ID": "467A8D7E-0DC6-F0C8-347C-BC4B315C397F",
+      "ActionKeyword": "at",
```
Comment (jjw24) [inline plugins.json:1627]:
```suggestion
```

File: plugins.json [hunk 2]
```diff
@@ -1621,5 +1621,19 @@
...
+      "Name": "AntdOpenBrowser",
+      "Description": "Quickly open the antd component",
+      "Author": "HenryTSZ",
+      "Version": "1.0.1",
+      "Language": "JavaScript",
+      "Website": "https://github.com/HenryTSZ/Flow.Launcher.AntdOpenBrowser",
+      "UrlDownload": "https://github.com/HenryTSZ/Flow.Launcher.AntdOpenBrowser/releases/download/v1.0.1/Flow.Launcher.Plugin.AntdOpenBrowser.zip",
+      "UrlSourceCode": "https://github.com/HenryTSZ/Flow.Launcher.AntdOpenBrowser/tree/main",
+      "ExecuteFileName": "./src/index.js",
```
Comment (jjw24) [inline plugins.json:1636]:
```suggestion
```

Comment (jjw24) [reply plugins.json:1636]: not needed

File: plugins.json (hunk 1 above)
Comment (jjw24) [reply plugins.json:1627]: not needed

File: plugins.json [hunk 3]
```diff
@@ -1621,5 +1621,19 @@
...
+      "IcoPath": "https://github.com/HenryTSZ/Flow.Launcher.AntdOpenBrowser/blob/main/app.png?raw=true"
```
Comment (jjw24) [inline plugins.json:1637]:
```suggestion
      "IcoPath": "https://cdn.jsdelivr.net/gh/HenryTSZ/Flow.Launcher.AntdOpenBrowser@main/app.png"
```

Comment (jjw24) [reply plugins.json:1637]: Please use CDN

**Comment by HenryTSZ** [comment]
What else do I need to do? Keep update branch?

**Comment by HenryTSZ** [comment]
Is adding a plugin so troublesome?

**Comment by VictoriousRaptor** [comment]
> Is adding a plugin so troublesome?

Sorry for merging so late. I think everyone's got buried by notifications.

**Review by VictoriousRaptor — APPROVED** [review]

**Pull request merged**